h/migrations/versions/64039842150a_add_user_comms_opt_in_column.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""Add User comms_opt_in column"""
+import sqlalchemy as sa
+from alembic import op
+
+revision = "64039842150a"
+down_revision = "3359d7d8ec29"
+
+
+def upgrade():
+    op.add_column("user", sa.Column("title", sa.UnicodeText()))
```

Comment:
```suggestion
    op.add_column("user", sa.Column("comms_opt_in", sa.UnicodeText()))
```

Reply:
We should probably get [autogeneration](https://alembic.sqlalchemy.org/en/latest/autogenerate.html) working to make these easier

Reply:
Oh FFS. This is embarrasing. I don't get how this happened (well I do, copy paste), but I tried this? It wouldn't work without it would it?

Reply:
This is now not totally random